osquery/CMakeLists.txt

```diff
@@ -29,13 +29,18 @@ else()
 endif()
 
 if(WIN32)
+  set(WINDOWS_DEP_DIR "$ENV{ChocolateyInstall}/lib")
+  string(REGEX REPLACE "\\\\" "/" WINDOWS_DEP_DIR ${WINDOWS_DEP_DIR})
   ADD_OSQUERY_LINK_CORE("netapi32.lib")
   ADD_OSQUERY_LINK_CORE("rpcrt4.lib")
   ADD_OSQUERY_LINK_CORE("shlwapi.lib")
   ADD_OSQUERY_LINK_CORE("wbemuuid.lib")
+  ADD_OSQUERY_LINK_CORE("linenoise.lib" "${WINDOWS_DEP_DIR}/linenoise-ng/local/lib")
```

Comment:
Can the install for these Chocolatey packages link their `./lib` directories into `$ENV{ChocolateyInstall}/lib`?